packages/skill-lesson/trpc/trpc-context.ts

```diff
@@ -1,8 +1,19 @@
 import {PrismaClient} from '@skillrecordings/database'
 import * as trpc from '@trpc/server'
 import * as trpcNext from '@trpc/server/adapters/next'
+import {NextApiRequest, NextApiResponse} from 'next'
+import {Session} from 'next-auth'
```

Comment:
this is what actually solves the issue, but explicitly defining the type below is less mystery meat